machine.go

```diff
@@ -115,32 +162,41 @@ type Machine struct {
 
 // Logger returns a logrus logger appropriate for logging hypervisor messages
 func (m *Machine) Logger() *log.Entry {
-	// return m.logger.WithField("subsystem", "firecracker-go-sdk")
-	return m.logger
+	return m.logger.WithField("subsystem", "firecracker-go-sdk")
```

Comment:
nit: Can you the const `userAgent` here?

Reply:
Sure

Reply:
Done